src/Microsoft.ML.Transforms/Text/WordEmbeddingsTransform.cs

```diff
@@ -324,9 +343,189 @@ public Mapper(WordEmbeddingsTransform parent, Schema inputSchema)
+                //                    |                        '-- Add --'                Cast (to = float)          |
+                //                    |   Initialize (D [k + 3, j]   |                            |                  |
+                //                    |             |                |                     Clip (min = 1.0)          |
+                //                    |             |----------------|----------------------------|--------\         |
+                //                    |             |                |                            |         \        |
+                //                    |   /---------'-------------\  |                            |          '----\  |
+                //                  Gather                        Gather                          |               Gather
+                //                    |                              |                            |                  |
+                //            ReduceMin (axes = [0])      ReduceMax (axes = [0])                  |        ReduceSum (axes = [0])
+                //                    |                              |                            |                  |
+                //                    |                              |                            '------- Div ------'
+                //                    |                              |                                      |
+                //                    |                              |                                      |
+                //                    '------------------- Concat (axis = 1) -------------------------------'
+                //                                                   |
+                //                                               P [j * 3]
+
+                long[] axes = new long[] { 0 };
+                // Allocate D, a constant tensor representing word embedding weights
+                var shapeD = new long[] { _parent._currentVocab.GetNumWords() + 3, _parent._currentVocab.Dimension };
+                var wordVectors = _parent._currentVocab.WordVectors;
+                var tensorD = new List<float>();
+                tensorD.AddRange(wordVectors);
+                tensorD.AddRange(Enumerable.Repeat(0.0f, _parent._currentVocab.Dimension));
+                tensorD.AddRange(Enumerable.Repeat(float.MaxValue, _parent._currentVocab.Dimension));
+                tensorD.AddRange(Enumerable.Repeat(float.MinValue, _parent._currentVocab.Dimension));
+                var nameD = ctx.AddInitializer(tensorD, shapeD, "WordEmbeddingWeights");
+
+                // Allocate F, a value representing an out-of-dictionary word
+                var tensorF = _parent._currentVocab.GetNumWords();
+                var nameF = ctx.AddInitializer(tensorF, "NotFoundValueComp");
+
+                // Retrieve X, name of input
+                var nameX = srcVariableName;
+
+                // Do label encoding
```

Comment:
```suggestion
                // Do label encoding. Out-of-vocab tokens will be mapped to the size of vocabulary. Because the index of vocabulary is zero-based, the size of vocabulary is the smallest upper bound of all vocabulary indexes computed by this LabelEncoder.
```